.cirrus.yml

```diff
@@ -1,29 +1,30 @@
 ---
-# We use Cirrus for Vagrant tests, because macOS instances of GHA
-# are too slow and flaky, and Linux instances of GHA do not support KVM.
+# We use Cirrus for Vagrant tests and native CentOS 7 and 8, because macOS
+# instances of GHA are too slow and flaky, and Linux instances of GHA do not
+# support KVM.
 
 # NOTE Cirrus execution environments lack a terminal, needed for
 # some integration tests. So we use `ssh -tt` command to fake a terminal.
 
-compute_engine_instance:
-  image_project: cirrus-images
-  image: family/docker-kvm
-  platform: linux
-  nested_virtualization: true
-  # CPU limit: `16 / NTASK`: see https://cirrus-ci.org/faq/#are-there-any-limits
-  cpu: 8
-  # Memory limit: `4GB * NCPU`
-  memory: 32G
+task:
+  name: ci / fedora 34 (vagrant)
```

Comment:
Can be not rename the test (it is currently called `vagrant DISTRO:fedora34`)? It is marked as required in this repo configuration, and we'll need to change that.

Or, if you're renaming it, maybe call it vagrant/fedora or something, so we would not have to change it when we switch to f35.

Reply:
Changed the name back to `vagrant DISTRO:fedora34`. I still kept the new names of the CentOS tests as they are no longer vagrant based.